docs/content/installation/csi_ug_requirements.md

```diff
@@ -37,33 +37,60 @@ Perform these steps for each worker node in Kubernetes cluster to prepare your e
 
    **Additional configuration steps for OpenShift® Container Platform users (RHEL and RHCOS).** Other users can continue to step 3.
 
-   Download and save the following yaml file:
+   Download and save the following YAML file:
 
    ```
    curl https://raw.githubusercontent.com/IBM/ibm-block-csi-operator/master/deploy/99-ibm-attach.yaml > 99-ibm-attach.yaml
    ```
 
    This file can be used for both Fibre Channel and iSCSI configurations. To support iSCSI, uncomment the last two lines in the file.
 
-   **Important:** The 99-ibm-attach.yaml configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this yaml file, as necessary.
+   **Important:** The `99-ibm-attach.yaml` configuration file overrides any files that already exist on your system. Only use this file if the files mentioned are not already created. <br />If one or more have been created, edit this YAML file, as necessary.
 
-   Apply the yaml file.
+   Apply the YAML file.
 
    `oc apply -f 99-ibm-attach.yaml`
+
+3. Configure storage system connectivity.
+
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
     
-3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
 
    For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
 
    Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
 
-   The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
+   The instructions and relevant YAML files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
-4. Configure storage system connectivity.
+5. (Optional) If planning on using volume replication (remote copy function), enable support on your Kubernetes cluster.
```

Comment:
in contrast to previous features, replication also requires one-time actions **on the storage side** in advance (as we saw in HyperSwap, which was removed from this version)

Reply:
specifically for svc, I know the `mkfcpartnership` is required, but @roysahar-ibm might have more details.
since it's different per storage system, we should probably have a separate page, was we did for finding the system id

Reply:
@roysahar-ibm please specify what exactly is required on the storage side currently for SVC so I can see how to properly add it into the documentation. 
for reference - [mkfcpartnership](https://www.ibm.com/docs/en/flashsystem-9x00/8.4.x?topic=commands-mkfcpartnership)

Reply:
@barakda might know too

Reply:
an example for mkfcpartnership command:
`svctask mkfcpartnership -backgroundcopyrate 50 -linkbandwidthmbits 250 Cluster_9.151.155.217`

Reply:
is there any official docs we can link to for the whole procedure?

Reply:
few links:
https://www.ibm.com/docs/en/flashsystem-7x00/8.2.x?topic=mirror-metro-global-partnerships
https://www.ibm.com/docs/en/flashsystem-7x00/8.2.x?topic=commands-mkfcpartnership
https://www.redbooks.ibm.com/redpapers/pdfs/redp5103.pdf

Reply:
@barakda  thank you - are these values the specific ones that we need or I can just link / reference these specific topics?
svctask mkfcpartnership -backgroundcopyrate 50 -linkbandwidthmbits 250 Cluster_9.151.155.217
i couldn't find anything with mkfcpartnership within the redbooks

@oriyarde  I suggest to add in another step (or substep)

> 
> (Optional) If planning on using volume replication (remote copy function), enable support on your storage system, using the mkfcpartnership command.
> For more information, see the following sections within your Spectrum Virtualize product, on [IBM Documentation](https://www.ibm.com/docs/en):
> - Command-line interface > Copy Service commands > mkfcpartnership 
> - Product overview > Technical overview > Copy Services functions > Metro Mirror and Global Mirror partnerships

I can add i a bullet also for the redbooks if @barakda  confirms
Also, if the specific input for the command is needed, I would just reword a little differently and add it in.
**Ori** please confirm that this is what we need and I'll add it in.

Reply:
I don't think these specific command values should be included in our docs.
@barakda are we sure `mkfcpartnership` is enough?
@rivkap as I mentioned, IMO, the `enable replication support on your storage system` should link to a separate page for all steps (and per storage system in the future)

Reply:
@barakda could you help adapting your example with our existing examples, with minimal parameters?
would something like: `svctask mkfcpartnership demo-system-id` work?

Reply:
regarding the layout right now, I don't think it makes sense to separate into another page, unless we do need to add in more than one command, and even then. It's more to prepare for the future than anything else. This flows more and is very minimal of a step to define.
Let's wait and see what @barakda  and @roysahar-ibm  say about the text and then I'll look at it again for layout

Reply:
After speaking to @roysahar-ibm I'll be navigating the user to https://www.ibm.com/docs/en/flashsystem-7x00/8.4.x?topic=mcs-managing-remote-copy-partnerships-1 for storage side setup

Reply:
sounds good to me, the page separation can wait, unless we can just link to an external page, in which case we won't need a separate internal page

Reply:
resolving for now